### PR TITLE
Update dropshadows to the latest guidelines 

### PR DESCRIPTION
**Pull request by lokesh-sagi125**
## Description
updating dropshadow according to the latest guidelines and replacing box-shadow eith appropriate dropshadow

#### updated the elemnts in the kds repo to use the dropshadows according to the latest guidelines and replace other elements like boxshadows with appropriate dropshadow.

Addresses #724

## Changelog

  - **Description:**  Updates dropshadows to the latest design guidelines
  - **Products impact:**  Visual
  - **Addresses:** https://github.com/learningequality/kolibri-design-system/issues/724
  - **Components:** All components with dropshadows
  - **Breaking:**  no
  - **Impacts a11y:**  no
  - **Guidance:** -

### CODE REVIEW

**Comment by MisRob** [comment]
Thanks @lokesh-sagi125. Did very brief skim and overall it seems to me it does what it's supposed to :) I will invite my colleagues for a full review. Meanwhile, please run `yarn lint-fix` and commit to resolve linting issues.

**Comment by lokesh-sagi125** [comment]
yes @MisRob  on it

**Comment by lokesh-sagi125** [comment]
hey , just ran yarn-fix and updated the change log, sorry for the inconvenience i am new to contributing

**Comment by MisRob** [comment]
@lokesh-sagi125 Sure, no problem at all, we're happy to provide guidance to new contributors :) Thanks!

**Comment by MisRob** [comment]
@lokesh-sagi125 would you be able to resolve the conflict? I think you updated the styling page but I also updated it as part of the writing styling guidance that you followed. I just merged that to `develop` and it caused conflict here. You can just accept changes incoming from develop in _docs/pages/styling/index.vue__ file.

**Comment by MisRob** [comment]
@lokesh-sagi125 would you be able to resolve the conflict? I think you updated the styling page but I also updated it as part of the writing styling guidance that you followed. I just merged that to `develop` and it caused conflict here. You can resolve by (1) merging the latest upstream `develop` branch to this branch and (2) accept changes incoming from the upstream `develop` in _docs/pages/styling/index.vue__ file.

**Comment by MisRob** [comment]
@lokesh-sagi125 would you be able to resolve the conflict? I think you updated the styling page but I also updated it as part of the writing styling guidance that you followed. I just merged [my PR](https://github.com/learningequality/kolibri-design-system/pull/723) to `develop` and it caused conflict here. You can resolve by (1) merging the latest upstream `develop` branch to this branch and (2) accept changes incoming from the upstream `develop` in _docs/pages/styling/index.vue__ file. As a result, your PR will contain no changes to _docs/pages/styling/index.vue_.

Thanks for taking care of the file too and sorry I have forgotten to mention this when you picked up the issue.

**Comment by lokesh-sagi125** [comment]
@MisRob done with including the changes, can you try merging again?

**Comment by MisRob** [comment]
I already merged my work @lokesh-sagi125, that's why you saw the conflict. But I see you resolved that now! Looking good, thanks.

**Comment by lokesh-sagi125** [comment]
hey @MisRob  is there sth wrong with the pr or can i move on to another issue?

**Comment by MisRob** [comment]
Hi @lokesh-sagi125, I don't think that anything is wrong :) Seems nobody got to do the review yet. You don't need to wait for a review to be finished to work on another task, so you're welcome to pick something.

**Review by akolson — CHANGES_REQUESTED** [review]
Hi @lokesh-sagi125! great work on the clean up. As opposed to commenting out the old code, It makes more sense to remove it altogether, as we wont need it :). Based on the acceptance criteria,
> However, the unused %dropshadow-...s shouldn’t be deleted from definitions.scss yet (they are still being referenced from some non-KDS components in the products).

we can safely delete them save for the aforementioned exception.

File: CHANGELOG.md
Comment (MisRob) [inline CHANGELOG.md:1]: Leaving a note here that before merging it'd be good to revert CHANGELOG.md changes since the new GH action will now update the CHANGELOG.md after merging this PR by pasting the changelog from the PR description. Otherwise we'd have duplicate entries.

**Comment by MisRob** [comment]
Thanks for following-up @lokesh-sagi125. cc @akolson will leave the final approve on you, just leaving a note that I already updated the changelog description in the PR.

File: yarn.lock
Comment (MisRob) [inline yarn.lock:1]: Would you revert this file @lokesh-sagi125?

**Review by akolson — CHANGES_REQUESTED** [review]
After the revert of the yarn.lock, we should be good with this. Leaving a blocker here to prevent accidental merge. But all is good. Thanks @lokesh-sagi125 for your contribution!

File: yarn.lock
Comment (lokesh-sagi125) [reply yarn.lock:1]: done

File: yarn.lock
Comment (akolson) [inline yarn.lock]: Please remove this added new line that is causing the diff in this file.

File: yarn.lock
Comment (akolson) [reply yarn.lock:1]: Ideally this file shouldn't be a part of the pr. Could you please do [this](https://github.com/learningequality/kolibri-design-system/pull/736/files#r1738037926) and the revert should be complete. Thanks again @lokesh-sagi125

**Comment by MisRob** [comment]
Hi @lokesh-sagi125, another way to formulate what we're trying to say with @akolson regarding `yarn.lock` is that when you go to the _Files_ tab in this pull request https://github.com/learningequality/kolibri-design-system/pull/736/files, there should be no `yarn.lock` file present at all. This would cause issues with dependencies.

This is a nice technique for removing file from a pull request https://stackoverflow.com/a/46571620

File: yarn.lock
Comment (akolson) [reply yarn.lock:1]: Hi @lokesh-sagi125  apologies on the back and forth, but this latest change would now cause breakages as  `yargs-parser...` dependency has been removed. Could you please do [this](https://github.com/learningequality/kolibri-design-system/pull/736#issuecomment-2320523084) as suggested by my colleague @MisRob

**Comment by lokesh-sagi125** [comment]
the yarn.lock is not getting exclude through git commands , let me re-open the pr , sorry for the inconvenience

**Comment by akolson** [comment]
@lokesh-sagi125 no need to go through the trouble, we'll fix this on our side. Your pull request almost ready for the merge. Re-opening and reverting the yarn.lock. Thanks Again!

**Comment by akolson** [comment]
@lokesh-sagi125 I have already made the revert and will be pushing shortly. :)

**Comment by lokesh-sagi125** [comment]
hey @akolson @MisRob   sorry i couldn't of much help while fixing the issue, i will make sure to keep these issues in mind for my future contributions , thank you for being supportive :) .

**Comment by MisRob** [comment]
No problem @lokesh-sagi125, it was just 1 file out of 13 ;)

However I'm sorry to say you've just closed it by deleting your fork, and we can't re-open in this case.

**Comment by lokesh-sagi125** [comment]
oh im really sorry about that ; i was on the wrong user let me reopen it

**Comment by MisRob** [comment]
@lokesh-sagi125 I am pretty sure that was accidental. Just let us know if you wish to open a new PR ;)

**Comment by MisRob** [comment]
No worries.

**Comment by MisRob** [comment]
Oh wait, I can perhaps reopen it now.

**Comment by akolson** [comment]
@lokesh-sagi125 in case it's much of trouble, please let me know as I can create a branch with your changes if necessary. Thanks

**Pull request closed without merging**
